Patterns/Italian/Italian-DateTime.yaml

```diff
@@ -134,336 +130,402 @@ DateExtractor5: !nestedRegex
   def: \b{DayRegex}\s*[/\\\-]\s*{MonthNumRegex}\s*[/\\\-]\s*{DateYearRegex}
   references: [ DayRegex, MonthNumRegex, DateYearRegex ]
 DateExtractor6: !nestedRegex
-  def: (?<=\b(on|in|at)\s+){MonthNumRegex}[\-\.]{DayRegex}\b
-  references: [ MonthNumRegex, DayRegex ]
+  def: (?<=\b(le|sur|sur l[ae])\s+){MonthNumRegex}[\-\.\/]{DayRegex}\b
+  references: [MonthNumRegex, DayRegex ]
 DateExtractor7: !nestedRegex
-  def: \b{MonthNumRegex}\s*/\s*{DayRegex}((\s+|\s*,\s*|\s+of\s+){DateYearRegex})?\b
-  references: [ MonthNumRegex, DayRegex, DateYearRegex ]
+  def: \b{DayRegex}\s*/\s*{MonthNumRegex}((\s+|\s*,\s*){DateYearRegex})?\b
+  references: [ DayRegex, MonthNumRegex, DateYearRegex ]
 DateExtractor8: !nestedRegex
-  def: (?<=\b(on|in|at)\s+){DayRegex}[\\\-]{MonthNumRegex}\b
+  def: (?<=\b(le)\s+){DayRegex}[\\\-]{MonthNumRegex}\b
   references: [ DayRegex, MonthNumRegex ]
 DateExtractor9: !nestedRegex
-  def: \b{DayRegex}\s*/\s*{MonthNumRegex}((\s+|\s*,\s*|\s+of\s+){DateYearRegex})?\b
+  def: \b{DayRegex}\s*/\s*{MonthNumRegex}((\s+|\s*,\s*){DateYearRegex})?\b
   references: [ DayRegex, MonthNumRegex, DateYearRegex ]
 DateExtractorA: !nestedRegex
   def: \b{DateYearRegex}\s*[/\\\-]\s*{MonthNumRegex}\s*[/\\\-]\s*{DayRegex}
   references: [ DateYearRegex, MonthNumRegex, DayRegex ]
 OfMonth: !nestedRegex
-  def: ^\s*of\s*{MonthRegex}
+  def: ^\s*di\s*{MonthRegex}
   references: [ MonthRegex ]
 MonthEnd: !nestedRegex
-  def: '{MonthRegex}\s*(the)?\s*$'
+  def: '{MonthRegex}\s*(il)?\s*$'
   references: [ MonthRegex ]
+WeekDayEnd: !nestedRegex
+  def: '{WeekDayRegex}\s*,?\s*$'
+  references: [ WeekDayRegex ]
 RangeUnitRegex: !simpleRegex
-  def: \b(?<unit>years|year|months|month|weeks|week)\b
-OclockRegex: !simpleRegex
```

Comment:
Same. If unnecessary, you can set them to a placeholder.

Reply:
This will help keep the code between languages more similar. And avoid nullpointerexceptions in base code that might try to use the regexes.

Reply:
Feel free to add Italian-specific regexes as needed though.